### ISSUE TITLE
Interactive interface, TypeError: Cannot read properties of null (reading 'IsLeaf')

### ISSUE THREAD

**Issue by FatherofEverest** [merenlab/anvio#1864]
Received the following error when trying to manually rotate a tree using "command + right click" in anvi-interactive. For reference, I was working on a remote server using the ssh -L 8080:localhost:8080 name@server and using the following command to display a phylogeny 

anvi-interactive -p phylogenomic-profile_g97.db \
                 -t $concatAminoacids-phylogenomic-tree.txt \
                 --title "Veillonella Phylogenomics" \
                 --manual 

Here is the output for the error: 

Anvi'o version: `7`
```
TypeError: Cannot read properties of null (reading 'IsLeaf')
    at Bins.RebuildIntersections (http://0.0.0.0:8080/app/js/bin.js?rand=958d35315089fc88714d65b4efa291d3:1060:26)
    at Drawer.draw (http://0.0.0.0:8080/app/js/drawer.js?rand=1f403c852c371f6c59037855584b3685:114:10)
    at m.fn.init.onShow (http://0.0.0.0:8080/app/js/main.js?rand=efb83ea2ad6e4744c7b52c9fa407bd81:1589:28)
    at HTMLDivElement.<anonymous> (http://0.0.0.0:8080/app/lib/bootstrap-waitingfor/build/bootstrap-waitingfor.js?rand=ede89d67dfdee2e042aedee199b16dea:134:22)
    at HTMLDivElement.dispatch (http://0.0.0.0:8080/app/lib/jquery/dist/jquery.min.js?rand=b8b24a28e7fb1445f985a52eca30a0f2:4:8549)
    at HTMLDivElement.r.handle (http://0.0.0.0:8080/app/lib/jquery/dist/jquery.min.js?rand=b8b24a28e7fb1445f985a52eca30a0f2:4:5252)
    at Object.trigger (http://0.0.0.0:8080/app/lib/jquery/dist/jquery.min.js?rand=b8b24a28e7fb1445f985a52eca30a0f2:4:7650)
    at HTMLDivElement.<anonymous> (http://0.0.0.0:8080/app/lib/jquery/dist/jquery.min.js?rand=b8b24a28e7fb1445f985a52eca30a0f2:4:15517)
    at Function.each (http://0.0.0.0:8080/app/lib/jquery/dist/jquery.min.js?rand=b8b24a28e7fb1445f985a52eca30a0f2:2:2975)
    at m.fn.init.each (http://0.0.0.0:8080/app/lib/jquery/dist/jquery.min.js?rand=b8b24a28e7fb1445f985a52eca30a0f2:2:835)```

**Comment by matthewlawrenceklein** [comment]
hi there @FatherofEverest - I'm unable to reproduce this error with my own test data, would you be able to share your dataset so that I can test locally? You can email files to me at `mlklein@uchicago.edu`. Cheers :)

**Comment by FatherofEverest** [comment]
hi @matthewlawrenceklein - I can do that later if we think it's necessary. In the meantime, I'd like to add that the error is only generated when there are bins in the collection. I can rotate the tree fine when there are no bins.

**Comment by meren** [comment]
Hi @FatherofEverest, 

I tried to reproduce this error using `--manual` mode and with and without bins in the tree. Everything seems to be working fine here. It is likely that this issue is resolved in anvi'o `v7.1`. If you don't have time/energy to update your anvi'o and try again, you can send us the file `$concatAminoacids-phylogenomic-tree.txt`, and we can try it using your file specifically to see if there is something to fix on our end.

Thanks for your patience,
Best wishes.

**Comment by matthewlawrenceklein** [comment]
Closing for now, please feel free to re-address if your issue is unresolved :)
